homeassistant/components/template/cover.py

```diff
@@ -227,13 +224,13 @@ def _update_state(self, result):
 
         if state in _VALID_STATES:
             if not self._position_template:
-                if state in ("true", STATE_OPEN):
+                if state in ("true", CoverState.OPEN):
```

Comment:
I don't think we should use the entity state constant here. This is not the entity state but the rendering of a template.

Please update `_VALID_STATES` too to use other constants.

Reply:
https://github.com/home-assistant/core/pull/127853